source/extensions/transport_sockets/tls/io_handle_bio.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "envoy/network/io_handle.h"
+
+#include "openssl/bio.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace TransportSockets {
+namespace Tls {
+
+// NOLINTNEXTLINE(readability-identifier-naming)
+const BIO_METHOD* BIO_s_io_handle(void);
+
+// NOLINTNEXTLINE(readability-identifier-naming)
+BIO* BIO_new_io_handle(Envoy::Network::IoHandle* io_handle);
```

Comment:
Would it be possible to add some comments for this factory method?

Reply:
Done!